.github/workflows/positron-windows-nightly.yml

```diff
@@ -28,11 +29,6 @@ jobs:
         with:
           node-version-file: .nvmrc
 
-      # - name: Install miniconda
```

Comment:
Did you mean to remove this entirely (since it's commented out)?

Reply:
Yes, the conda install caused lots of issues, so I commented it out originally when testing and forgot to remove it.  Once everything else is working, we can revisit if it's worth trying to add back in.